app/extensions/torrent/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
yeah, i only needed it so i could run `Add Unpacked Extension` in chrome and get an ID for the new extension

i'll delete it